iron-overlay-manager.html

```diff
@@ -78,10 +78,58 @@
     },
 
     /**
+     * Brings the overlay at the specified index to the front.
+     * @param {number|Element} i
+     */
+    bringToFront: function(i) {
```

Comment:
Overall, I think it's most hygienic not to define APIs that accept dynamic types unless the type is not important to the implementation (e.g., you don't need `typeof` or other reflection features).

I'm happy if you make this private, but also I would prefer if we stick to one type for now unless explicitly needed.